src/index.js

```diff
@@ -332,6 +333,8 @@ function addSlotToRoot (root, slot) {
   }
 
   rootToSlotMap.get(root)[slotName] = slot;
+  !slotToRootMap.has(slot) && slotToRootMap.set(slot, root);
```

Comment:
I disagree about short guards, but agree with the idea of similar coding styles.
@treshugart how do you feel about adding atlassian eslint rules to skate build?